### PR TITLE
PHP 8 Support

### PR DESCRIPTION
**Pull request by eduardostuart**

### CODE REVIEW

**Comment by eduardostuart** [comment]
@googlebot I signed it!

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "sort-packages": true
   },
   "require": {
-    "php": "^5.5 || ^7.0",
+    "php": "^5.5|^7.0|^8.0",
```
Comment (jfschmakeit) [inline composer.json:26]: I believe we can simplify this to `>=5.5` at this point (similar to `goog/gax`.)

**Review by jfschmakeit — CHANGES_REQUESTED** [review]
Thanks for preparing this!

Can you also confirm that all tests successfully pass on PHP 8?

**Review by ryihan — CHANGES_REQUESTED** [review]
Change Update 8.0

**Comment by skysphr** [comment]
Why wasn't this merged?

**Pull request closed without merging**